python/tskit/trees.py

```diff
@@ -3850,6 +3863,43 @@ def trees(
         )
         return TreeIterator(tree)
 
+    def coiterate(self, other, **kwargs):
```

Comment:
Actually, might make more sense to put this next to the ``trees`` iterator in the file? It's nice to keep methods somewhat grouped by topic.

Reply:
Erm. That's where it is, isn't it?

Reply:
Yep, so it is. GitHub's way of showing context is a bit weird sometimes